### ISSUE TITLE
Parse changelog details for WordPress plugins served by WPackagist

### ISSUE THREAD

**Issue by danielbachhuber** [dependencies-io/support#63]
It would be nice if Dependencies.io pulled out the changelog details for WordPress plugins served by WPackaglist.

Here's how User Switching currently looks:

![image](https://user-images.githubusercontent.com/36432/52220945-e96b5480-2854-11e9-90c8-34d626046751.png)

This particular plugin has releases in its GitHub repo: https://github.com/johnbillion/user-switching/releases

Most plugins put their changelog in `readme.txt`: https://github.com/Yoast/wordpress-seo/blob/418370662e4bd8850eba09b2883cf49f05d6b71a/readme.txt#L109

**Comment by davegaeddert** [comment]
Sorry I'm slow to respond here. Can you give me a link to a composer.json that uses WPackaglist?

**Comment by danielbachhuber** [comment]
> Can you give me a link to a composer.json that uses WPackaglist?

Not sure I follow, can you clarify?

**Comment by davegaeddert** [comment]
Just looking for a practical example of how WPackagist is used and how the composer.json is different (you have to define that, right?). Now I'm looking at https://wpackagist.org/ and maybe I can just use the example on their homepage.

**Comment by danielbachhuber** [comment]
> maybe I can just use the example on their homepage.

Yep, the homepage example is a good one. If you have specific questions about the implementation, feel free to pull me into a technical discussion.
